src/screens/EmpowerPlant.tsx

```diff
@@ -124,45 +125,44 @@ export const selectImage = (source: string): React.ReactElement => {
   switch (image) {
     case 'plant-spider-cropped.jpg':
       return (
-        <Image
+        <ProfiledImage
           style={styles.tinyImage}
           source={require('../assets/images/plant-spider-cropped.png')}
         />
       );
     case 'plant-to-text-cropped.jpg':
       return (
-        <Image
+        <ProfiledImage
           style={styles.tinyImage}
           source={require('../assets/images/plant-to-text-cropped.png')}
         />
       );
     case 'nodes-cropped.jpg':
       return (
-        <Image
+        <ProfiledImage
           style={styles.tinyImage}
           source={require('../assets/images/nodes-cropped.png')}
         />
       );
     case 'mood-planter-cropped.png':
       return (
-        <Image
+        <ProfiledImage
           style={styles.tinyImage}
           source={require('../assets/images/mood-planter-cropped.png')}
         />
       );
     default:
       return (
-        <Image
+        <ProfiledImage
           style={styles.tinyImage}
           source={require('../assets/images/mood-planter-cropped.png')}
         />
       );
   }
 };
 
-/* You could wrap this with the Sentry Profiler, 
-* but then you'd have hundreds/thousands of spans because the tools response is not paginated.
-*/
+const ProfiledImage = Sentry.withProfiler(Image);
```

Comment:
This helps me remember the Profiler was used, as I go through the rest of this file. Sweet.